test/access_token_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class AccessTokenTest < Test::Unit::TestCase
+
+  def test_delegate_access_token
+    fake "access_tokens/delegate.json?delegate_access_scope%5B%5D=write_orders&" \
+      "delegate_access_scope%5B%5D=read_products&expires_in=", :method => :post,
+      :status => 201, :body => load_fixture('access_token_delegate'), :extension => false
```

Comment:
Never mind, I think the new syntax was introduced in 1.9 so I believe I can use the new syntax here.